fulp_modules/devils/devil.dm

```diff
@@ -0,0 +1,382 @@
+
+	for(var/obj/item/I in held_items) // Left hand items
+		if(!(I.item_flags & ABSTRACT))
+			. += "It is holding [I.get_examine_string(user)] in its [get_held_index_name(get_held_index_of_item(I))]."
+
+	if(!client && stat != DEAD) // Braindead
+		. += "The devil seems to be in deep contemplation."
+
+	if(stat == DEAD) // Damaged
+		. += "<span class='deadsay'>The hellfire seems to have been extinguished, for now at least.</span>"
+	else if(health < (maxHealth/10))
+		. += "<span class='warning'>You can see hellfire inside its gaping wounds.</span>"
+	else if(health < (maxHealth/2))
+		. += "<span class='warning'>You can see hellfire inside its wounds.</span>"
+	. += "*---------*</span>"
+
+/mob/living/carbon/true_devil/resist_buckle()
+	if(buckled)
+		buckled.user_unbuckle_mob(src,src)
+		visible_message("<span class='warning'>[src] easily breaks out of [p_their()] handcuffs!</span>", \
+					"<span class='notice'>With just a thought your handcuffs fall off.</span>")
+
+/mob/living/carbon/true_devil/canUseTopic(atom/movable/M, be_close=FALSE, no_dexterity=FALSE, no_tk=FALSE)
+	if(incapacitated())
+		to_chat(src, "<span class='warning'>You can't do that right now!</span>")
+		return FALSE
+	if(be_close && !in_range(M, src))
+		to_chat(src, "<span class='warning'>You are too far away!</span>")
+		return FALSE
+	return TRUE
+
+/// Beepsky doesn't like Devils
+/mob/living/carbon/true_devil/assess_threat(judgement_criteria, lasercolor = "", datum/callback/weaponcheck=null)
+	return 666
+
+/mob/living/carbon/true_devil/soundbang_act()
+	return FALSE
+
+/mob/living/carbon/true_devil/get_ear_protection()
+	return 2
+
+/mob/living/carbon/true_devil/attacked_by(obj/item/I, mob/living/user, def_zone)
+	apply_damage(I.force, I.damtype, def_zone)
+	var/message_verb = ""
+	if(length(I.attack_verb_continuous))
+		message_verb = "[pick(I.attack_verb_continuous)]"
+	else if(I.force)
+		message_verb = "attacked"
+
+	var/attack_message = "[src] has been [message_verb] with [I]."
+	if(user)
+		user.do_attack_animation(src)
+		if(user in viewers(src, null))
+			attack_message = "[user] has [message_verb] [src] with [I]!"
+	if(message_verb)
+		visible_message("<span class='danger'>[attack_message]</span>",
+		"<span class='userdanger'>[attack_message]</span>", null, COMBAT_MESSAGE_RANGE)
+	return TRUE
+
+/mob/living/carbon/true_devil/can_be_revived()
+	return TRUE
+
+/// They're immune to fire.
+/mob/living/carbon/true_devil/resist()
+	set name = "Resist"
+	set category = "IC"
+
+	if(!can_resist())
+		return
+	changeNext_move(CLICK_CD_RESIST)
+
+	SEND_SIGNAL(src, COMSIG_LIVING_RESIST, src)
+	if(!HAS_TRAIT(src, TRAIT_RESTRAINED) && pulledby) // Resisting Grabs
+		log_combat(src, pulledby, "resisted grab")
+		resist_grab()
+		return
+
+	if(on_fire)
+		resist_fire() // Resisting out of Fires
+
+/// Intents
+/mob/living/carbon/true_devil/attack_hand(mob/living/carbon/human/M)
+	. = ..()
+	if(.)
+		switch(M.a_intent)
+			if("harm")
+				var/damage = rand(1, 5)
```

Comment:
see, this kind of shit is why I went "uh oh" when I saw that you made true devils carbons but not humanoids

a beefman's punch is gonna do the same amount of damage to a devil as a plasteel golem's punch will. I mean, yeah, that's true for simplemobs and shit as well, but it's still kind of a bruh moment